### ISSUE TITLE
`addaveragingerror` doesn't actually do anything due to functionality in `get_obs_surface`

### ISSUE THREAD

**Issue by brendan-m-murphy** [openghg/openghg_inversions#42]
TLDR: `get_obs_surface` already performs the calculations in `addaveragingerror` and resamples the data, so `addaveraging` error doesn't actually change anything.

If an `averaging` argument is passed to `openghg.retrieve.get_obs_surface`, then the following happens:

1. The obs data is resampled to the given resampling period:
```
ds_resampled = data.resample(time=average).mean(skipna=False, keep_attrs=True)
```
2. If `ch4_repeatability` (or `..._repeatability`) is present, then in the resampled data, `ch4_repeatability` is set to the variance of the mean of the original repeatability values:
```
if "repeatability" in var:
                ds_resampled[var] = (
                    np.sqrt((data[var] ** 2).resample(time=average).sum())
                    / data[var].resample(time=average).count()
                )

```
3. A variable called `f{species}_repeatability` (e.g. `ch4_repeatability`) is added, which contains the standard deviation of the observations over the resampling period:
```
ds_resampled[f"{species}_variability"] = (
      data[species].resample(time=average).std(skipna=False, keep_attrs=True)
)
```

Now, `addaveragingerror` in `openghg_inversions` does the following:
1. gets the obs again, with the same averaging period as specified in `fixedbasisMCMC` or `get_data.py`:
```
 get_obs = get_obs_surface(site=site,
                                  species=species,
                                  inlet=inlet[i],
                                  instrument=instrument[i],
                                  average=meas_period[i],
                                  start_date=start_date,
                                  end_date=end_date)
```
2. converts the obs values to a data frame called `sitedataerr` and pads the date range with NaNs: I'm not totally sure why this step is necessary
3. If `mf_variability` is present (which will always be the case if `meas_period[i]` is not `None`), it sets the `mf_variability` data variable of `fp_all[site]` to
```
np.sqrt(fp_all[site.upper()].mf_variability.values**2 + sitedataerr.resample(meas_period[i]).std(ddof=0).dropna().values.T**2))
```
Since the data was resampled to `meas_period[i]` by `get_obs_surface`, the second term in the square root is 0 (since the standard deviation of one value is zero), so `mf_variability` isn't changed.
4. The same is done for `mf_repeatability`: the `mf_repeatability` of `fp_all[site]` (if it is present) is set to
```
np.squeeze(np.sqrt(fp_all[site.upper()].mf_repeatability.values**2 + sitedataerr.resample(meas_period[i]).std(ddof=0).dropna().values.T**2))
```
but again, the second term in the `np.sqrt` is zero.

What this *should* be doing is adding the `mf_variability` to the `mf_repeatability` if `mf_repeatability` is present. This doesn't require fetching the data again.
